openBEM.js

```diff
@@ -2908,7 +2890,7 @@ function annual_solar_rad(region, orient, p) {
     return 0.024 * sum;
 }
 
-
+// This function is no longer actually used anywhere...
```

Comment:
Tempted to delete this, but thought I'd just point out that it's currently redundant.